src/components/ui/menu/menu.component.tsx

```diff
@@ -149,7 +149,7 @@ export class Menu extends React.Component<MenuProps> {
     const descriptor = this.service.createDescriptorForElement(info.item, info.index);
     const selected: boolean = this.isItemSelected(descriptor);
 
-    return this.cloneItemWithProps(info.item, { descriptor, selected, onPress: this.onItemPress });
+    return this.cloneItemWithProps(info.item, { descriptor, selected, onItemPress: this.onItemPress });
```

Comment:
Instead of defining a special prop for that, I would suggest leaving only onPress, but with the following structure:

`onPress: (event, descriptor)`

This way we save the original React way of handling press events by leaving event object as first argument passed to handler function. But we still able to use internal implementation (descriptor) as a second argument.

```
private renderItem = (info: ListRenderItemInfo<MenuItemElement>): React.ReactElement => {
    const descriptor = this.service.createDescriptorForElement(info.item, info.index);

    return this.cloneItemWithProps(info.item, {
      descriptor: descriptor,
      selected: this.isItemSelected(descriptor),
      onPress: (event: GestureResponderEvent): void => {
        info.item.props.onPress && info.item.props.onPress(event, descriptor);
        this.onItemPress(descriptor);
      },
    });
  };
```